### ISSUE TITLE
Kafka Connector - in different way - feature proposal

### ISSUE THREAD

**Issue by us3r** [prestodb/presto#12327]
At the moment kafka connector is using `Kafka Simple Consumer`. This approach have some pros and cons. It is easy to use - but from my point of view it make sense to use it only for small topic or just for some kind of healthchecks (simple review of the topic content, etc). 
It would be perfect if we could create more advanced version of this conenctor.

If we could define in the session:

```
set session kafka.group_id = prod-consumer;
set session kafka.auto_offset_reset = earliest;
```
and then
`select count(*) from kafka.schema.topic`

will return all messages that we have in the kafka, like `100`

Next run of this query will give us:
```
select count(*) from kafka.schema.topic;
2
```

So thats mean that only 2 new events were appeard  in the kafka from last check.

```
set session kafka.group_id = prod-consumer2;
set session kafka.auto_offset_reset = earliest ;
select count(*) from kafka.schema.topic;
102
```

So in other words it would be perfect if we could add support for kafka consumer groups :)
